AdblockPlusSafari/NSAttributedString+TextRenderer.h

```diff
@@ -22,7 +22,7 @@
 @interface NSAttributedString (TextRenderer)
 
 /**
- @param token a single character representing a span marker.
+ @param markerChar a single character representing a span marker.
```

Comment:
Should surely be fixed, but doesn't seem related to issue 3184, so please do it in a separate commit. `Noissue` is fine.